src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1015,7 +1015,7 @@
     {
       "run": "after",
       "snippetPath": "statisticalCodeTypesReferenceCheckTrigger.sql",
-      "fromModuleVersion": "23.0.0"
+      "fromModuleVersion": "25.0.0"
```

Comment:
Yes, I think this is the change needed on the mainline (assuming the next version is 25.0.0 rather than 24.1.0).

A different version will be needed for the back port.